docs/sphinx/user_guide/feature/view.rst

```diff
@@ -330,6 +330,61 @@ indices by :math:`N`, ::
     }
   }
 
+Index Layout
+^^^^^^^^^^^^
+
+``RAJA::IndexLayout`` is a layout that can use an index list to map
+input indices to an entry within a view.  Each dimension of the layout
+can have its own indexing strategy to determine this mapping.
+
+Three indexing strategies are natively supported in RAJA:
+DirectIndex, IndexList, and ConditionalIndexList. DirectIndex maps an input

```

Comment:
```suggestion
``RAJA::DirectIndex``, ``RAJA::IndexList``, and ``RAJA::ConditionalIndexList``. ``DirectIndex`` maps an input

```

To make it clear that the index types are in the RAJA namespace.  After that, you don't need to keep repeating the namespace, but format the type names so they are highlighted similarly to other types in the docs.